Added missing descriptions (1657156). (#19563)
dotnet/docs: docs/framework/migration-guide/application-compatibility.md

---
title: Runtime and retargeting changes - .NET Framework

> description: Learn about application compatibility in the .NET Framework and how it is affected by runtime and retargeting changes when migrating to another version.

Reviewer: ```suggestion
description: Learn about application compatibility in .NET Framework and how it's affected by runtime and retargeting changes when migrating to another version.
```
